src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md

```diff
@@ -0,0 +1,265 @@
+| --------------------- | ------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name (string)         | Optional. The name of the main script file, so something like "script.js".                       | The name of the test in the k6 Cloud UI. Test runs with the same name will be grouped together.                                                                                                                                     |
+| projectID (number)         | Optional. It is empty by default.                       | The ID of the project in which the test is assigned in the k6 Cloud UI. By default, the default project of the user default organization.                                                                                                                                       |
+| distribution (object) | Optional. The equivalent of `someDefaultLabel: { loadZone: "amazon:us:ashburn", percent: 100 }`. | How the traffic should be distributed. The keys are string labels that will be injected as [environment variables](/using-k6/environment-variables) on the appropriate nodes (matching the `loadZone`): `__ENV["someDefaultLabel"]` |
+
+### List of supported load zones
+
+| Location              | ID                    |
+| --------------------- | --------------------- |
+| Tokyo                 | `amazon:jp:tokyo`     |
+| Seoul                 | `amazon:kr:seoul`     |
+| Mumbai                | `amazon:in:mumbai`    |
+| Singapore             | `amazon:sg:singapore` |
+| Sydney                | `amazon:au:sydney`    |
+| Montreal              | `amazon:ca:montreal`  |
+| Frankfurt             | `amazon:de:frankfurt` |
+| Ireland               | `amazon:ie:dublin`    |
+| London                | `amazon:gb:london`    |
+| Paris                 | `amazon:fr:paris`     |
+| Stockholm             | `amazon:se:stockholm` |
+| N. Virginia (Default) | `amazon:us:ashburn`   |
+| Ohio                  | `amazon:us:columbus`  |
+| N. California         | `amazon:us:palo alto` |
+| Oregon                | `amazon:us:portland`  |
+| Hong Kong             | `amazon:cn:hong kong` |
+
+
+### Running tests under a different project than your default one
+
+By default tests and test runs will be created and run under your default project, in your default organization.
+
+To create and run tests under a different project, whether under your default organization or one you've been invited to, you have to pass the `Project ID` to k6.
+
+Select the project on the sidebar menu and you will find the `Project ID` on the header of the Project Dashboard page.
```

Comment:
"**on** the header" => "**in** the header"

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720